block_iagora.php

```diff
@@ -66,12 +100,13 @@ public function get_content() {
      * @return string The generated HTML content for the chat.
      */
     private function generate_chat_content($copilotendpointurl) {
-        global $OUTPUT;
+        global $OUTPUT, $USER, $COURSE;
         // Generate a unique identifier for the chat container.
         $chatid = uniqid('iagora_chat_');
         $context = [
             'chatId' => $chatid,
             'tokenEndpointURL' => $copilotendpointurl,
+            'completedactivities' => json_encode($this->get_student_progress()),
```

Comment:
```suggestion
            'completedActivities' => json_encode($this->get_student_progress()),
```
Aren't JavaScript variables using camelCase?

Reply:
yes, but moodle has not accepted this naming.Here's the error he made  _Variable "completedActivities" must be all lower-case_